.ci/check-changelogs.groovy

```diff
@@ -34,7 +34,8 @@ pipeline {
       }
       steps {
         deleteDir()
-        gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true)
+        gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true,
+                    depth: 3, reference: "/var/lib/jenkins/.git-references/${REPO}.git")
```

Comment:
How does the git reference repo get cached?
- [Job](https://infra-ci.elastic.co/view/Packer/job/elastic+infra+master+packer-builder-apm-ci-linux/)
- [latest build](https://infra-ci.elastic.co/job/elastic+infra+master+packer-builder-apm-ci-linux/BUILDER=gcp,BUILD_OS=ubuntu-1604-lts,label=linux%20&&%20immutable%20&&%20docker/lastBuild/consoleFull)

```
INFO:     googlecompute: TASK [git_fetch_reference : clone reference repos] *****************************
...
INFO:     googlecompute: changed: [default] => (item=git@github.com:elastic/apm-server.git)```